app/models/concerns/symphony_request.rb

```diff
@@ -6,7 +6,7 @@ def appears_in_myaccount?
   end
 
   def send_to_symphony!(options = {})
-    SubmitSymphonyRequestJob.perform_now(self, options)
+    SubmitSymphonyRequestJob.perform_later(id, options)
```

Comment:
I wrote that code before the comment regarding mediated item approval, so yes, this has to change.